CMakeLists.txt

```diff
@@ -25,9 +24,14 @@ set(CHRONOLOG_PACKAGE_TARNAME "${CHRONOLOG_PACKAGE}")
 #------------------------------------------------------------------------------
 # Setup install and output Directories
 #------------------------------------------------------------------------------
-#if(NOT CMAKE_INSTALL_PREFIX)
-set(CMAKE_INSTALL_PREFIX /home/$ENV{USER}/chronolog)
-#endif()
+# Validate CMAKE_BUILD_TYPE
+if(NOT (CMAKE_BUILD_TYPE STREQUAL "Debug" OR CMAKE_BUILD_TYPE STREQUAL "Release"))
```

Comment:
please, add checks for CMAKE_INSTALL_PREFIX and CMAKE_BUILD_TYPE being defined

if(NOT DEFINED CMAKE_INSTALL_PREFIX )
   set (CMAKE_INSTALL_PREFIX...)
endif()

and 
if (NOT DEFINED CMAKE_BUILD_TYPE OR
      CMAKE_BUILD_TYPE STREQUEAL ...